packages/test/test-end-to-end-tests/src/test/summaryDataStoreStats.spec.ts

```diff
@@ -98,7 +98,7 @@ describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
 			if (
 				event.eventName === "fluid:telemetry:Summarizer:Running:Summarize_generate" &&
 				event.referenceSequenceNumber
-					? event.referenceSequenceNumber >= sequenceNumber
+					? (event.referenceSequenceNumber as number) >= sequenceNumber
```

Comment:
'ITelemetryBaseEvent' is a map-like object with unknown keys/values, hence 'referenceSequenceNumber' is typed to 'string | number | a bunch of other stuff...'.

TS5 no longer allows comparison between non-numeric types, hence the new cast is required.